tests/puppeteer/pages/commonPage.js

```diff
@@ -52,13 +52,14 @@ module.exports = class CommonPage {
    * @param selector, where to click
    * @return newPage, what was opened by the browser
    */
-  async openLinkWithTargetBlank(currentPage, selector) {
-    const pageTarget = await currentPage.target();
-    await currentPage.click(selector);
-    const newTarget = await global.browser.waitForTarget(target => target.opener() === pageTarget);
-    this.page = await newTarget.page();
-    await this.page.waitForSelector('body');
-    return this.page;
+  async openLinkWithTargetBlank(currentPage, selector, waitForNavigation = true) {
+    const [newPage] = await Promise.all([
+      new Promise(resolve => this.page.once('popup', resolve)),
+      currentPage.click(selector),
+    ]);
+    if (waitForNavigation) await newPage.waitForNavigation({waitUntil: 'networkidle0'});
```

Comment:
Is there some cases we *don't* want to wait for navigation ?

Reply:
Yes, after install when it click on go to FO, browser don't detect navigation.